code/def_files/data/effects/main-f.sdr

```diff
@@ -280,11 +280,11 @@ void main()
 	}
 	if(gammaSpec) {
 		specColor.rgb = max(specColor.rgb, vec3(0.03f)); // hardcoded minimum specular value. read John Hable's blog post titled 'Everything Is Shiny'. 
- #ifdef FLAG_HDR
-		specColor.rgb = srgb_to_linear(specColor.rgb);
- #endif
 		fresnelFactor = 1.0;
 	}
+	#ifdef FLAG_HDR
+		specColor.rgb = srgb_to_linear(specColor.rgb);
```

Comment:
The indentation is not correct here. It should be one level below since this code is not in the if body anymore.